### ISSUE TITLE
special.orthogonal.he_roots() gives halved weights in 0.15 compared to 0.14

### ISSUE THREAD

**Issue by hmgaudecker** [scipy/scipy#4683]
Is this expected behaviour? I could not find anything in the changelog, my excuses if I missed it.

```
In [15]: import scipy

In [16]: scipy.__version__
Out[16]: '0.14.0'

In [17]: n, w = scipy.special.orthogonal.he_roots(3)

In [18]: w
Out[18]: array([ 0.41777138,  1.67108552,  0.41777138])

In [19]: w.sum()
Out[19]: 2.5066282746310002
```

```
In [7]: import scipy

In [8]: scipy.__version__
Out[8]: '0.15.1'

In [9]: n, w = scipy.special.orthogonal.he_roots(3)

In [10]: w
Out[10]: array([ 0.20888569,  0.83554276,  0.20888569])

In [11]: w.sum()
Out[11]: 1.2533141373155003

In [12]: w.sum() * 2
Out[12]: 2.5066282746310007
```

**Comment by pv** [comment]
@ewmoore: ping, gh-3914

**Comment by ewmoore** [comment]
Yes, this is my error.  `mu` should be `sqrt(2*pi)` instead of `sqrt(pi)`.

Incidentally,  
```
In [19]: integrate.quad(special.hermitenorm(0).weight_func, -np.inf, np.inf)
Out[19]: (3.5449077018110327, 1.7716827031171878e-08)
```
Is also wrong.  (And probably has been forever I'd guess.  Before my time anyway.)

**Comment by ewmoore** [comment]
Fix in #4684.
